cnspec-lint/action.yaml

```diff
@@ -0,0 +1,26 @@
+name: "Lint cnspec policies"
+author: "mondoohq"
+description: "This action runs `cnspec bundle lint` and outputs the results as Sarif report so that the results can easily be shows in GitHub code scanning results"
```

Comment:
```suggestion
description: "This action runs `cnspec bundle lint` and outputs the results as SARIF report so that the results can easily be shown in GitHub code scanning results"
```